src/Formatter/General.php

```diff
@@ -74,9 +72,10 @@ public function setLogEntryPrefix($logEntryPrefix)
      * Format the event into a message string.
      *
      * @param array $event
-     * @return array|mixed|string
+     * @psalm-suppress ImplementedReturnTypeMismatch
+     * @return string
      */
-    public function format($event)
+    public function format($event) // @phpstan-ignore-line
```

Comment:
can this ignore line if needed be in PHPdoc above?

Reply:
IT should be really easy, I just need the reference to the rule we're ignoring. Unfortunately I can't find a list of phpstan rules like you get for eslint.

Reply:
will it not work simply by moving it to the PHPdoc above without changes? Having specific ignore would be nice but if its difficult, i am good with having ignore-line

Reply:
No, PHPStan lints it's own ignore comments and was complaining that the rule was missing. I've sorted it now though.